#задание 4
def func(a, b, c):
    try:
        result = sum(a, b, c)
        return result
    except:
        return 'error'
#я делала, но забыла закоммитить
